xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-test/xwiki-platform-mentions-test-docker/src/test/it/org/xwiki/mentions/test/ui/MentionsIT.java

```diff
@@ -191,6 +191,51 @@ void comment(TestUtils setup, TestReference reference) throws Exception
         });
     }
 
+    /**
+     * This test is similar to {@link #comment(TestUtils, TestReference)} but this time the comment is created on a page
+     * with an empty title. In this case, the link to the page displayed in the mention must have the name of the
+     * document displayed instead of an empty title.
```

Comment:
Why do you need an IT test? Couldn't you have a simple unit test that makes sur that the right title is computed instead, even when there's no title?

Reply:
I didn't find a simpler method to assert that, but if it exists I'm interested to learn it.